storm-core/src/ui/public/flux.html

```diff
@@ -0,0 +1,139 @@
+<html>
```

Comment:
Not sure which is better between F (for Flux) and Y (for Yaml). I guess the question could be changed to is flux yaml viewer **flux** yaml viewer, or flux **yaml** viewer. What do you think?